src/components/Home/Home.tsx

```diff
@@ -0,0 +1,49 @@
+import "./Home.css";
+import { useDispatch, useSelector } from "react-redux";
+import { UserStore } from "../../redux/store";
+import fallbackImg from "../../images/fallback_img.jpg";
+import { logOut } from "../../redux/form/formAction";
+
+function Home() {
+  const dispatch = useDispatch();
+  const user = useSelector((state: UserStore) => state.userDetail);
+  const name = user?.name;
+  const email = user?.email;
+  const phoneNo = user?.phone;
+  const ImageUrl = user?.photo ? user.photo : "";
```

Comment:
[Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) can be used instead of ternary.